### PR TITLE
Add course level properties to qe

### PR DESCRIPTION
**Pull request by AlexBedley**
Just adding the properties, not doing anything with them yet

backend: https://git.dev.d2l/projects/CORE/repos/lms/pull-requests/6806/overview

### CODE REVIEW

File: components/d2l-quick-eval/d2l-quick-eval.js [hunk 1]
```diff
@@ -132,6 +132,16 @@ class D2LQuickEval extends
 			returningToQuickEval: {
 				type: Boolean,
 				value: false
+			},
+			courseLevel: {
+				type: Boolean,
+				value: false
+			},
+			submissionFilters: {
+				type: Array
```
Comment (dmackenz) [inline components/d2l-quick-eval/d2l-quick-eval.js]: Do these arrays need a default value?

Comment (cbusmand2l) [reply components/d2l-quick-eval/d2l-quick-eval.js]: I would say that we don't need a default yet until we know what's coming back from the API (do we know that yet?). If `[]` is a valid response that we want to differentiate against no response/error/not making a request, then undefined as our default value might make sense. We can add in a default (if necessary) once we know how we want to deal with it.

Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval.js]: Hmm good point Carol. To me empty array `[]` suggests filtering is disabled, but `undefined` suggests maybe a version mismatch or something, which in that case maybe we want a default set of filters on the front end? Or maybe we should blow up on `undefined`? I'm not sure tbh

Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval.js]: Ok I'm gonna set the default to `[]` for now, and we can revert if we change our minds

File: components/d2l-quick-eval/d2l-quick-eval.js [hunk 2]
```diff
@@ -60,16 +60,27 @@ class D2LQuickEval extends
...
+			<d2l-quick-eval-submissions
+				href="[[_lazySubmissionsHref]]"
+				token="[[token]]"
+				logging-endpoint="[[loggingEndpoint]]"
+				data-telemetry-endpoint="[[dataTelemetryEndpoint]]"
+				hidden$="[[_displayActivities(toggleState, activitiesViewEnabled)]]"
+				master-teacher="[[masterTeacher]]"
+				returning-to-quick-eval="[[returningToQuickEval]]"
+				course-level="[[courseLevel]]"
+				submission-filters="[[submissionFilters]]"></d2l-quick-eval-submissions>
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval.js]: I just reformatted these to be on a new line for ease-of-use

**Comment by AlexBedley** [comment]
Since I'm doing this to avoid merge conflicts I figured I might as well pass them down. I tested this against a real LMS and found a bug with the filters (they aren't stringifying properly). But `courseLevel` works fine.

**Pull request merged**